protocol/avdl/chat1/common.avdl

```diff
@@ -34,6 +34,7 @@ protocol common {
   }
 
   record RateLimit {
+    string name;
```

Comment:
Rate limit names are stored as strings in the database, so this is consistent with that.